libnetwork/portallocator/portallocator_windows.go

```diff
@@ -1,9 +1,11 @@
 package portallocator
 
-func init() {
+const (
```

Comment:
This was really weird code; the default was set in a non-platform specific file, and then a (windows-only) `init()` to overwrite the defaults.